inc/lift/RequestHandle.h

```diff
@@ -79,6 +80,17 @@ class RequestHandle
         std::chrono::milliseconds timeout
     ) -> bool;
 
+    /**
+     * Sets the maximum number of bytes of data to write.
+     * (The number of bytes downloaded may be greater than the set amount,
+     * but the number of bytes written for the response's data will not
+     * exceed this amount.)
```

Comment:
you should mention here how to set it to "infinite" or "the full file"

Reply:
Added! Should I do anything to make it more prominent?